### PR TITLE
HPCC-12798 Remove auto discovery option

### PR DESCRIPTION
**Pull request by rpastrana**
- Removes auto and manual ip option
- Update verbiage

Signed-off-by: Rodrigo Pastrana <rodrigo.pastrana@lexisnexis.com>

## Type of change:
- [x] This change is a bug fix (non-breaking change which fixes an issue).
- [ ] This change is a new feature (non-breaking change which adds functionality).
- [ ] This change is a breaking change (fix or feature that will cause existing behavior to change).
- [ ] This change alters the query API (existing queries will have to be recompiled)

## Checklist:
- [x] My code follows the code style of this project.
  - [ ] My code does not create any new warnings from compiler, build system, or lint.
- [x] The commit message is properly formatted and free of typos.
  - [x] The commit message title makes sense in a changelog, by itself.
  - [x] The commit is signed.
- [x] My change requires a change to the documentation.
  - [ ] I have updated the documentation accordingly, or...
  - [x] I have created a JIRA ticket to update the documentation.
  - [ ] Any new interfaces or exported functions are appropriately commented.
- [x] I have read the CONTRIBUTORS document.
- [x] The change has been fully tested:
  - [ ] I have added tests to cover my changes.
  - [ ] All new and existing tests passed.
  - [ ] I have checked that this change does not introduce memory leaks.
  - [ ] I have used Valgrind or similar tools to check for potential issues.
- [x] I have given due consideration to all of the following potential concerns:
  - [ ] Scalability
  - [ ] Performance
  - [ ] Security
  - [ ] Thread-safety
  - [ ] Premature optimization
  - [ ] Existing deployed queries will not be broken
  - [x] This change fixes the problem, not just the symptom
  - [x] The target branch of this pull request is appropriate for such a change.
- [x] There are no similar instances of the same problem that should be addressed
  - [ ] I have addressed them here
  - [ ] I have raised JIRA issues to address them separately
- [x] This is a user interface / front-end modification
  - [x] I have tested my changes in multiple modern browers
  - [x] The component(s) render as expected

### CODE REVIEW

**Comment by rpastrana** [comment]
@wangkx please review UI
@JamesDeFabia please review verbiage

File: esp/files/configmgr.html [hunk 1]
```diff
@@ -518,21 +518,14 @@
              <div class="bd noborders" style="background-color: White;border:none">
                <h1>Environment setup</h1>
                <h2>Welcome to wizard mode!</h2>
-                Define IP Addresses for the environment being configured. Choose manual entry to enter IP addresses or 
-                auto discovery to acquire the list of IP Addresses via auto discovery script.
-                Manual entry format:X.X.X.X; X.X.X.X-XXX;
+                Define IP Addresses, or hostnames for the environment being configured.
```
Comment (JamesDeFabia) [inline esp/files/configmgr.html]: No comma needed

**Review by JamesDeFabia — CHANGES_REQUESTED** [review]
Verbiage is fine except for the superfluous comma

**Comment by richardkchapman** [comment]
@rpastrana Back to you. Also can you fill in the checkboxes please

**Comment by wangkx** [comment]
@rpastrana The UI changes look fine.

**Pull request merged**